schemas/data_catalogue/b18/energy_scan/current.json

```diff
@@ -65,13 +96,14 @@
                     "title": "Mirror Types",
                     "type": "object",
                     "properties": {
-                        "collimating_mirror": {
-                            "title": "Collimating Mirror",
+                        "double_crystal_monochromator": {
+                            "title": "Double Crystal Monochromator",
                             "enum": [
                                 "311",
                                 "111"
                             ],
-                            "type": "string"
+                            "type": "string",
+                            "nexus_path": ["/entry1/instrument/DoubleCrystal/dcm_crystal"]
```

Comment:
It looks like the actual NeXus values here are Si(111) and I guess Si(311) would be the other